lib/datadog/tracing/contrib/http/integration.rb

```diff
@@ -1,11 +1,11 @@
 # typed: false
 
-require 'datadog/tracing/contrib/configuration/resolvers/pattern_resolver'
-require 'datadog/tracing/contrib/http/circuit_breaker'
-require 'datadog/tracing/contrib/http/configuration/settings'
-require 'datadog/tracing/contrib/http/patcher'
-require 'datadog/tracing/contrib/integration'
-require 'ddtrace/version'
+require_relative '../configuration/resolvers/pattern_resolver'
+require_relative 'circuit_breaker'
+require_relative 'configuration/settings'
+require_relative 'patcher'
+require_relative '../integration'
+require_relative '../../../../ddtrace/version'
 
 module Datadog
   module Tracing
```

Comment:
Minor: ⬇️ The `MINIMUM_VERSION` logic here makes no sense at all. The `.version` is also incorrect, seeing as nowadays `net-http` can even be installed and updated separately from Ruby (https://rubygems.org/gems/net-http).

Reply:
I'll capture this in an issue.

Reply:
https://github.com/DataDog/dd-trace-rb/issues/2184